packages/lesswrong/lib/collections/moderatorActions/helpers.ts

```diff
@@ -102,35 +103,48 @@ type ReasonReviewIsNeeded = "mapLocation"|"firstPost"|"firstComment"|"contactedT
 type GetReasonForReviewResult =
     { needsReview: false, reason: ReasonNoReviewNeeded }
   | { needsReview: true, reason: ReasonReviewIsNeeded }
+type ReasonForInitialReview = Exclude<ReasonReviewIsNeeded, 'newContent'> 
+const reviewReasonsSetting = 
+  new DatabasePublicSetting<Array<ReasonForInitialReview>>(
+    'moderation.reasonsForInitialReview', 
+    ['firstPost', 'firstComment', 'contactedTooManyUsers', 'bio', 'profileImage'])
 
 export function getReasonForReview(user: DbUser|SunshineUsersList): GetReasonForReviewResult
 {
-  const fullyReviewed = user.reviewedByUserId && !user.snoozedUntilContentCount;
   /**
    * This covers several cases
    * 1) never reviewed users
    * 2) users who were removed from the review queue and weren't previously reviewed
    * 3) users who were removed from the review queue and *were* previously reviewed
    * 1 & 2 look indistinguishable, 3 will have a non-null reviewedAt date
-   */ 
-  const unreviewed = !user.reviewedByUserId;
-  const snoozed = user.reviewedByUserId && user.snoozedUntilContentCount;
+   */
 
+  const fullyReviewed = user.reviewedByUserId && !user.snoozedUntilContentCount;
   if (fullyReviewed) {
     return {needsReview: false, reason: 'alreadyApproved'};
   }
+  
+  const unreviewed = !user.reviewedByUserId;
+  const snoozed = user.reviewedByUserId && user.snoozedUntilContentCount;
+  
+  const reviewReasonMap: Record<ReasonForInitialReview, Function> = {
```

Comment:
```suggestion
  const reviewReasonMap: Record<ReasonForInitialReview, ()=>boolean> = {
```

And add `!!`s on the return values. Seems generally a bit nicer, and would have stopped me from being a moron when I put in a console log while testing.